packages/docusaurus-plugin-content-docs/src/theme/DocSidebar/styles.module.css

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@media (min-width: 997px) {
+  .sidebar {
+    height: calc(100vh - var(--ifm-navbar-height));
+    padding: 0.5rem;
+    position: sticky;
```

Comment:
On PC narrow screen, it takes on the `position: fixed` from `.menu.menu--show` when the sidebar is displaying. This is causing the scrollbar from the content to show up on Firefox and Safari.

Reply:
Not sure how impactful this is because the scenario can be relative rare 🙈

Reply:
@wgao19 can you elaborate on this issue? Anyway I was combining the CSS modules and the Infima classes, which is not recommended (another example why it's not good!)

Reply:
Good catch, D1 handles this issue well. I'll fix it in Infima 😄